pyaedt/common_rpc.py

```diff
@@ -253,7 +255,7 @@ def launch_server(port=18000, ansysem_path=None, non_graphical=False, threaded=T
 
 def create_session(server_name, client_port=None, launch_aedt_on_server=False, aedt_port=None, non_graphical=True):
     """
-    Connect to an existing AEDT server session.
+    Connect to an existing AEDT server session and connect to it.
```

Comment:
This changed line confuses me. The method says "create_session" and this talks about connecting to an existing session (plus it talks about connecting twice).